ibis/tests/all/test_vectorized_udf.py

```diff
@@ -10,9 +10,22 @@ def add_one(s):
     return s + 1
 
 
+@elementwise(input_type=[dt.double], output_type=[dt.double])
+def add_one_with_output_type_list(s):
```

Comment:
I think you can inline this function in the tests, we don't think we need to use this in other tests.